src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,221 @@
+package seedu.duke;
+
+import java.util.List;
+
+/**
+ * Holds the interaction between user and program.
+ */
+public class Ui {
+
+    /**
+     * Default constructor.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Prints welcome words to screen.
+     */
+    public void welcomeWord() {
+        System.out.println("//////////\n" + "->Hello! I'm Duke\n" +
+                "->What can I do for you?" + "\n");
```

Comment:
Should there be a break before the operator?

Reply:
Yeah, I have fixed it but haven't pushed to github😂